Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -805,10 +870,20 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
         NSDictionary *oldUserInfo = [error userInfo];
         NSMutableDictionary *newUserInfo = oldUserInfo ? [NSMutableDictionary dictionaryWithDictionary:oldUserInfo] : [NSMutableDictionary dictionary];
         newUserInfo[@"networkTaskDescription"] = [task description];
+        if (auto* networkDataTask = [self existingTask:task]) {
```

Comment:
Diff looks wrong, no? (see two lines down)

Reply:
Fixed — it looks like I applied the diff incorrectly when I rebased, oops.

Thanks for the review!